### ISSUE TITLE
add "box-sizing: border-box" to ReactDropdownSelect

### ISSUE THREAD

**Issue by karland** [sanusart/react-dropdown-select#94]
I cannot PR because I am getting an error. Here is the problem:

![grafik](https://user-images.githubusercontent.com/4803370/81596919-22ce3080-93c5-11ea-88bc-63f89c4dd0f9.png)

This is the solution:

![grafik](https://user-images.githubusercontent.com/4803370/81596560-9459af00-93c4-11ea-925f-60599d1861e8.png)

Thanks for the library. Nicely done.

**Comment by sanusart** [comment]
Thanks, need to check carefully how will it affect the layout of current usage.

**Comment by karland** [comment]
Ok, fine. Thanks.

**Comment by sanusart** [comment]
@karland I don't see this happening if I paste:

```jsx
<div style={{display: 'flex', alignItems: 'center'}}>
{'From:'} 
<Select
  options={options}
  values={[]}
  onChange={(value) => console.log(value)}
/> 
{'to:'} 
<Select
  options={options}
  values={[]}
  onChange={(value) => console.log(value)}
/>
</div>
```

Into live edit of the example at https://sanusart.github.io/react-dropdown-select/examples

![image](https://user-images.githubusercontent.com/267718/83947443-970ebf00-a81f-11ea-9c2f-0329547657fd.png)

**Comment by karland** [comment]
Well, my suspicion is that you have something like

```CSS
* {
  box-sizing: border-box;
}
```

in global css. I hadn't, because the default css is [`box-sizing: content-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

**Comment by sanusart** [comment]
Fixed in v4.4.2, Thanks!

**Comment by karland** [comment]
Thanks.
